python/ray/data/read_api.py

```diff
@@ -1047,14 +1047,32 @@ def read_tfrecords(
            length  width    species
         0     5.1    3.5  b'setosa'
 
+        We can also read compressed TFRecord files which uses one of the
+        `compression type supported by Arrow <https://arrow.apache.org/docs/python/generated/pyarrow.CompressedInputStream.html>`:
```

Comment:
I think you miss a `_` at the end, see other example [here](https://github.com/ray-project/ray/blob/master/python/ray/data/read_api.py#L577).